.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+Fixes # .
```

Comment:
Most in anecdotal and I'd disagree. Very commonly do they address a ticket, and we usually just fail to link it. If it doesnt address a ticket, I think its good to ask "why?".

This is also just a template, if it doesnt address a ticket that blurb can be removed.